lib/Test/Pod/CoverageChange.pm

```diff
@@ -0,0 +1,250 @@
+use Pod::Coverage;
+use File::Find::Rule;
+use Test::Pod::Coverage;
+use Module::Path qw(module_path);
+use List::Util qw(any);
+
+use constant {
+    POD_SYNTAX_IS_OK => 0,
+    FILE_HAS_NO_POD  => -1,
+};
+
+use Exporter qw(import export_to_level);
+our @EXPORT_OK = qw(pod_coverage_syntax_ok);
+
+=head2 pod_coverage_syntax_ok
+
+Checks all the modules under a given directory against POD coverage and POD syntax
+
+=over 4
+
+=item * C<$path> - path or arrayref of directories to check (recursively)
+
+example: ['lib', 'other directory'] | 'lib'
+
+=item * C<$allowed_naked_packages> - hashref of number of allowed naked subs, keyed by package name (optional)
+
+example: {Package1 => 2, Package2 => 1, Package3 => 10}
+
+=item * C<$ignored_packages> - arrayref of packages that will be ignored in the checks (optional)
+
+example: ['MyPackage1', 'MyPackage2', 'MyPackage3']
+
+=back
+
+=cut
+
+sub pod_coverage_syntax_ok {
+    my $path                   = shift;
+    my $allowed_naked_packages = shift // {};
+    my $ignored_packages       = shift // [];
+
+    $path             = [$path]             unless ref $path eq 'ARRAY';
+    $ignored_packages = [$ignored_packages] unless ref $ignored_packages eq 'ARRAY';
+
+    check_pod_coverage($path, $allowed_naked_packages, $ignored_packages);
+    check_pod_syntax($path, $ignored_packages);
+}
+
+=head2 check_pod_coverage
+
+Checks POD coverage for all the modules that exists under a given directory.
+Passes the C<$allowed_naked_packages> to L<Test::Pod::CoverageChange/check_allowed_naked_packages>.
+Ignores the packages in the C<$ignored_packages> parameter.
+
+=over 4
+
+=item * C<$path> - path or arrayref of directories to check (recursively)
+
+=item * C<$allowed_naked_packages> - hashref of number of allowed naked subs, keyed by package name (optional)
+
+=item * C<$ignored_packages> - arrayref of packages that will be ignored in the checks (optional)
+
+=back
+
+=cut
+
+sub check_pod_coverage {
+    my $path                   = shift;
+    my $allowed_naked_packages = shift;
+    my $ignored_packages       = shift;
+
+    check_allowed_naked_packages($allowed_naked_packages, $ignored_packages) if keys %$allowed_naked_packages;
+
+    # Check for newly added packages PODs
+    my @ignored_packages = (keys %$allowed_naked_packages, @$ignored_packages);
+    foreach my $package (Test::Pod::Coverage::all_modules(@$path)) {
+        next if any { $_ eq $package } @ignored_packages;
+        pod_coverage_ok($package, {private => []});
+    }
+}
+
+=head2 check_pod_syntax
+
+Check POD syntax for all the modules that exists under a given directory.
+
+=over 4
+
+=item * C<$path> - path or arrayref of directories to check (recursively)
+
+=item * C<$ignored_packages> - arrayref of packages that will be ignored in the checks (optional)
+
+=back
+
+=cut
+
+sub check_pod_syntax {
+    my $path             = shift;
+    my $ignored_packages = shift;
+    my $Test_Builder     = Test::More->builder;
```

Comment:
lower case?

Reply:
I looked at some examples that used this module and I figured it out It's common to write test builder variable like this, I prefer to follow the conventions. Also IMO it's more readable for who wanted to read the code.